### PR TITLE
Fix double click on auto-test submit button

### PR DESCRIPTION
**Pull request by ninox14**
**🟢 Add `deploy` label if you want to deploy this Pull Request to staging environment**

#### 🧑‍⚖️ Pull Request Naming Convention

- Title should follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- Do not put issue id in title
- Do not put WIP in title. Use [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) functionality
- Consider to add `area:*` label(s)

* [ ] I followed naming convention rules

---

#### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Test Case
- [ ] Documentation update
- [ ] Other

#### 🔗 Related issue link

#2177 

#### 💡 Background and solution

_Describe the big picture of your changes here_

#### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Database migration is added or not needed
- [x] Documentation is updated/provided or not needed
- [x] Changes are tested locally

### CODE REVIEW

File: client/src/modules/AutoTest/components/Exercise/Exercise.tsx [hunk 1]
```diff
@@ -48,7 +48,7 @@ function Exercise({ githubId, courseId, courseTask, finishTask }: ExerciseProps)
         <Form form={form} layout="vertical" requiredMark={false} onFinish={submit} onChange={change}>
           {getExercise()}
           <Row justify="center">
-            <Button loading={loading} type="primary" htmlType="submit">
+            <Button loading={loading} type="primary" htmlType="submit" disabled={loading}>
```
Comment (valerydluski) [inline client/src/modules/AutoTest/components/Exercise/Exercise.tsx:51]: this does not fix the problem

Comment (ninox14) [reply client/src/modules/AutoTest/components/Exercise/Exercise.tsx:51]: I just retested that, and this fixes at least the rapid clicking issue
[video](https://pomf2.lain.la/f/prdctm1m.mp4)
maybe I'm missing something
I will look into it more if needed

File: client/src/modules/AutoTest/hooks/useCourseTaskSubmit/useCourseTaskSubmit.ts [hunk 2]
```diff
@@ -109,6 +109,10 @@ export function useCourseTaskSubmit(courseId: number, courseTask: CourseTaskVeri
   };
 
   const submit = async (values: FormValues) => {
+    if (loading) {
+      return;
+    }
```
Comment (AlreadyBored) [inline client/src/modules/AutoTest/hooks/useCourseTaskSubmit/useCourseTaskSubmit.ts:114]: Looks excessive :)

Comment (ninox14) [reply client/src/modules/AutoTest/hooks/useCourseTaskSubmit/useCourseTaskSubmit.ts:114]: Can you please elaborate on that?

**Review by aaliakseyenka — APPROVED** [review]

**Pull request merged**